components/localize/localize-mixin.js

```diff
@@ -0,0 +1,166 @@
+import IntlMessageFormat from 'intl-messageformat/src/main.js';
+window.IntlMessageFormat = IntlMessageFormat;
+
+export const D2LLocalizeMixin = superclass => class extends superclass {
+
+	static get properties() {
+		return {
+			language: {type: String},
+			langResources: { type: Object },
+			formats: {
```

Comment:
We could also exclude this for now unless we run into a case where we need to allow for custom formats.